ietf/group/views.py

```diff
@@ -1276,7 +1276,10 @@ def stream_documents(request, acronym):
     editable = has_role(request.user, "Secretariat") or group.has_role(request.user, "chair")
     stream = StreamName.objects.get(slug=acronym)
 
-    qs = Document.objects.filter(states__type="draft", states__slug__in=["active", "rfc"], stream=acronym)
+    qs = Document.objects.filter(stream=acronym).filter(
+        Q(type_id="draft", states__type="draft", states__slug="active")
+        | Q(type_id="rfc")
```

Comment:
Would it make sense to specify the "published" state here in case we add additional rfc states?

Reply:
I made a different adjustment to switch on doc type instead. Result makes the search intent more obvious